src/main/resources/MiLoG_Template.tex

```diff
@@ -3,241 +3,252 @@
-			& \tableentry{2}{!break} 
-			& \tableentry{2}{!dayTotal}\\
+			!action
+			& !date
+			& !begin
+			& !end
+			& !break
+			& !dayTotal\\
 			\hline
 			%Line 22
-			\tableentry{6}{!action}
-			& \tableentry{2}{!date} 
-			& \tableentry{2}{!begin} 
-			& \tableentry{2}{!end} 
-			& \tableentry{2}{!break} 
-			& \tableentry{2}{!dayTotal}\\
+			!action
+			& !date
+			& !begin
+			& !end
+			& !break
+			& !dayTotal\\
 			\hline 
 			%Leerzeile
 			\multicolumn{6}{c}{\thinspace}\\
 			%Urlaub
-			\cline{4-6}
-			\multicolumn{3}{c}{\thinspace} 
-			& \multicolumn{2}{|c|}{\centering Urlaub anteilig:}
-			& \tableentry{2}{!vacation}\\
-			\cline{4-6}
+			\cline{3-6}
+			\multicolumn{2}{c}{\thinspace}
+			& \multicolumn{3}{|c|}{\centering \textbf{Urlaub anteilig:}}
+			& !vacation\\
+			\cline{3-6}
 			%Summe
-			\multicolumn{3}{c}{\thinspace} 
-			& \multicolumn{2}{|c|}{\centering Summe:}
-			& \tableentry{2}{!sum}\\
-			\cline{4-6}
+			\multicolumn{2}{c}{\thinspace}
+			& \multicolumn{3}{|c|}{\centering \textbf{Summe:}}
+			& !sum\\
+			\cline{3-6}
 			%SollArbeitszeit
-			\multicolumn{3}{c}{\thinspace} 
-			& \multicolumn{2}{|c|}{\centering monatliche Soll-Arbeitszeit:}
-			& \tableentry{2}{!workingTime}\\
-			\cline{4-6}
+			\multicolumn{2}{c}{\thinspace}
+			& \multicolumn{3}{|c|}{\centering \textbf{monatliche Soll-Arbeitszeit:}}
+			& !workingTime\\
+			\cline{3-6}
 			%Übertrag Vormonat
-			\multicolumn{3}{c}{\thinspace} 
-			& \multicolumn{2}{|c|}{\centering Übertrag vom Vormonat:}
-			& \tableentry{2}{!carryPred}\\
-			\cline{4-6}
+			\multicolumn{2}{c}{\thinspace}
+			& \multicolumn{3}{|c|}{\centering \textbf{Übertrag vom Vormonat:}}
+			& !carryPred\\
+			\cline{3-6}
 			%Übertrag Folgemonat
-			\multicolumn{3}{c}{\thinspace} 
-			& \multicolumn{2}{|c|}{\centering Übertrag in den Folgemonat:}
-			& \tableentry{2}{!carrySucc}\\
-			\cline{4-6}
-		\end{tabular}	
+			\multicolumn{2}{c}{\thinspace}
+			& \multicolumn{3}{|c|}{\centering \textbf{Übertrag in den Folgemonat:}}
+			& !carrySucc\\
+			\cline{3-6}
+		\end{tabular}
 	\end{center}
-	
+
 	%FOOTER
-	\par \bigskip \bigskip \bigskip
+	\par \bigskip \bigskip \medskip
 	\headentry{\large Ich bestätige die Richtigkeit der Angaben: \hspace*{\fill} $\overline{{\parbox{5.75cm}{\normalsize Datum, Beschäftigte/r} } }$ } \par \medskip
-	\headentry{\normalsize Nach $\S 17$ Mindestlohngesetz (MiLoG) müssen für geringfügig entlohnte und kurzfristig beschäftigte Arbeitnehmer/innen u.a. Beginn, Ende und Dauer der täglichen Arbeitszeit aufgezeichnet und für Kontrollzwecke mindestens 2 Jahre am Ort der Beschäftigung aufbewahrt werden!} \par \bigskip \bigskip
+	\headentry{\normalsize Nach \textbf{$\S 17$ Mindestlohngesetz (MiLoG)} müssen für geringfügig entlohnte und kurzfristig beschäftigte Arbeitnehmer/innen u.a. Beginn, Ende und Dauer der täglichen Arbeitszeit aufgezeichnet und für Kontrollzwecke mindestens 2 Jahre am Ort der Beschäftigung aufbewahrt werden!} \par \bigskip \bigskip
```

Comment:
I see a little problem here with printing math sections bold. It does not work at the moment but is not a big problem either.

Reply:
True, but what was the reason to put the § inside a math environment in the first place? (It seems the font we use does not support printing a bold § though.)

Reply:
The main reason was that the math environment always supports the §, whereas some TeX text environments, e.g. the one on my computer (not sure what one it is), does not compile the § without warning and therefore in one single step.
We wanted to avoid such errors and warnings to make the compile process as cross-compatible as possible.

As I said, no big deal :) It just rushed into my mind, when i was reading this line initially.

Reply:
Oh, I see. Should we print the number 17 bold though (moving it out of the math section)? Or would that look confusing given the § in front of it is not bold?

Reply:
Looks ok too, I think.
![image](https://user-images.githubusercontent.com/56889322/70154944-a581ef80-16b1-11ea-99f4-ea3c8bd40e73.png)

Reply:
So, which one do you think looks better? :D

Reply:
Not easy to make a choice on that. I would prefer the new, bold alternative because it looks a little more uniform.